.github/workflows/ci.yml

```diff
@@ -579,16 +578,16 @@ jobs:
     environment: release
 
     steps:
-    - uses: actions/checkout@v2
```

Comment:
`v4` is the latest major version of [`actions/checkout`](https://github.com/actions/checkout) at the time of this writing.